### ISSUE TITLE
Error applying option 'blend' to filter 'tvai_up': Option not found

### ISSUE THREAD

**Issue by MapleGateway** [Vargol/ffmpeg-apple-arm64-build#18]
I'm very new to all this so apologies in advance if I've goofed something.

Using this export command from Topaz:
```shell
ffmpeg "-hide_banner" "-i" "in.mkv" "-sws_flags" "spline+accurate_rnd+full_chroma_int" "-color_trc" "1" "-colorspace" "6" "-color_primaries" "6" "-filter_complex" "tvai_up=model=prob-3:scale=0:w=1620:h=1080:preblur=0:noise=0.7:details=0.7:halo=0:blur=-1:compression=0:estimate=20:blend=1:device=0:vram=1:instances=1,scale=w=1620:h=1080:flags=lanczos:threads=0" "-c:v" "tiff" "-pix_fmt" "rgb48le" "-compression_algo" "deflate" "-start_number" "1" "out/%06d.tiff"
``` 
I get this error:
```shell
Error applying option 'blend' to filter 'tvai_up': Option not found
```
If I remove blend=1 it does work but I do need blend.
If I specify the path to Topaz's ffmpeg, I don't get the error and it works as expected, but I'd like to use this ffmpeg build.

Any ideas?

**Comment by Vargol** [comment]
Hi. I'm not no in a position to test anything at quiet this moment.  
Can you check the build.sh script is using the latest version of the Topaz redistribution , it should be 6.0.1.9.

```
echoSection "compile ffmpeg with topaz"
$SCRIPT_DIR/build-ffmpeg-topaz.sh "$SCRIPT_DIR" "$WORKING_DIR" "$TOOL_DIR" "$OUT_DIR" "$CPUS" "6.0.1.9" > "$WORKING_DIR/build-ffmpeg-topaz.log" 2>&1
else
```

**Comment by MapleGateway** [comment]
Fixed!
TIL about commits... I had just downloaded the 3.0 release but I see you just released 3.1 with this fixed as well

Thank you very much!
